### PR TITLE
updating Microsoft SQL integration including contributor edits

### PR DESCRIPTION
**Pull request by akristen**
This PR is a duplicate from this PR https://github.com/newrelic/docs-website/pull/17681 

It contains edits and new sample code snippets to the set up procedures. I've created a new PR since the initial branch was from June. I'm also going to restructure this doc to follow our OHI patterns, which shouldn't take more than a day. Goal is to have it merged by Thursday/before end of week. 

cc: @phillipsj

### CODE REVIEW

**Comment by akristen** [comment]
@phillipsj: Looks like this doc exists in an updated format over in https://docs.newrelic.com/install/mysql. Have your updates been reflected in this newer doc?

**Comment by phillipsj** [comment]
@akristen that is for MySQL, not MSSQL. Those are two different things.

**Comment by akristen** [comment]
> @akristen that is for MySQL, not MSSQL. Those are two different things.

Oh, I sent you the wrong link. This is the correct one:  https://docs.newrelic.com/install/microsoft-sql/

I'm going to work with one of our SMEs about this integration and see if they can also find alignment between the two.

**Comment by akristen** [comment]
hi again @phillipsj -- had to deal with some version drift. Merged your edits into our "source of truth" for this integration. Want to make sure that your additions about gMSA are reflected the way you want them to be here: https://deploy-preview-18745--docs-website-netlify.netlify.app/install/microsoft-sql/

File: src/install/microsoft-sql/intro.mdx [hunk 1]
```diff
@@ -1,18 +1,19 @@
 ---
 headingText: Before we start
 componentType: default
+redirects:
+    - /docs/infrastructure/host-integrations/host-integrations-list/microsoft-sql/microsoft-sql-server-integration
+    - /docs/infrastructure/host-integrations/host-integrations-list/microsoft-sql/microsoft-sql-server-config
 ---
 
-Our Microsoft SQL Server [integration](/docs/integrations/host-integrations/getting-started/introduction-host-integrations) collects and sends inventory and metrics from your MS SQL Server environment to our platform, where you can see the health of your MS SQL Server environment. We collect both database and instance-level metrics so that you can pinpoint the source of any problems.
+Our Microsoft SQL Server integration collects and sends metric and inventory data from your Microsoft SQL Server environment to our platform so you can monitor the health of your Microsoft SQL Server environment. We collect both database and instance-level metrics so you can troubleshoot and resolve performance problems.
```
Comment (akristen) [inline src/install/microsoft-sql/intro.mdx:9]:
```suggestion
Our Microsoft SQL Server integration collects and sends metric and inventory data from your Microsoft SQL Server environment to our platform. We collect both database and instance-level metrics so you can monitor the health of your environment, then troubleshoot and resolve performance problems.
```

File: src/install/microsoft-sql/install-infra.mdx [hunk 2]
```diff
@@ -3,4 +3,8 @@ componentType: default
 headingText: Install the infrastructure agent
 ---
 
-To use the Microsoft SQL Server integration, you need to first [install the infrastructure agent](/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent-new-relic/) on the same host. The infrastructure agent monitors the host itself, while the integration you'll install in the next step extends your monitoring with Microsoft SQL Server-specific data.
+To use the Microsoft SQL Server integration, you need to first [install the infrastructure agent](/docs/infrastructure/infrastructure-monitoring/get-started/choose-infra-install-method) on a Linux or Windows host, or on a host capable of remotely accessing Microsoft SQL Server. The infrastructure agent monitors the host itself, while the integration you'll install in the next step extends your monitoring with Microsoft SQL Server-specific data.
+
+<Callout variant="important">
+    If using a gMSA for SQL Server access, then the infrastructure agent needs to run as the gMSA.
```
Comment (akristen) [inline src/install/microsoft-sql/install-infra.mdx:9]:
```suggestion
    If using a gMSA for Microsoft SQL Server access, then the infrastructure agent needs to run as the gMSA.
```

**Review by paperclypse — DISMISSED** [review]
Looks good. Thanks for making these fixes!

**Review by paperclypse — APPROVED** [review]
Thanks for swapping those!

**Pull request merged**